src/components/VxpMultiSelectDropdown/NativeSelectionModal.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <Frame>
```

Comment:
we can move base or helper components into core folder

Reply:
moved